Marlin/Marlin_main.cpp

```diff
@@ -457,7 +457,7 @@ static int serial_count = 0;
 // Inactivity shutdown
 millis_t previous_cmd_ms = 0;
 static millis_t max_inactive_time = 0;
-static millis_t stepper_inactive_time = (DEFAULT_STEPPER_DEACTIVE_TIME) * 1000UL;
+millis_t stepper_inactive_time = (DEFAULT_STEPPER_DEACTIVE_TIME) * 1000UL;
 
```

Comment:
Please reread about the static keyword.
In this case static limits the knowledge about `stepper_inactive_time` to inside the `Marlin_main.cpp ` file. Not more and not less.
http://www.cprogramming.com/tutorial/statickeyword.html case 3.